server/repository/src/db_diesel/invoice_row.rs

```diff
@@ -49,7 +49,11 @@ joinable!(invoice -> user_account (user_id));
 pub enum InvoiceRowType {
     OutboundShipment,
     InboundShipment,
-    InventoryAdjustment,
+    // Initially we had single inventory adjustment InvoiceRowType, this was changed to two seperate types
+    // central server may have old inventory adjustement type, thus map it to inventory additions
+    #[serde(alias = "INVENTORY_ADJUSTMENT")]
```

Comment:
Doing this in sync translation layer would require a lot more work (another invoice row type with manual mapping), not sure happy that it's in repository layer though, I wish we could auto derive things for  other crates within workspace